analyses/molecular-subtyping-HGG/09-HGG-with-braf-clustering.Rmd

```diff
@@ -0,0 +1,417 @@
+---
+title: "HGAT samples without histone mutations that have `BRAF V600E` mutations"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Chante Bethell for ALSF CCDL
+date: 2020
+---
+
+This notebook will look at HGAT samples without histone mutations that have `BRAF V600E` mutations.
+
+The purpose is to identify samples that may need to be reclassified as LGAT instead of HGAT using t-SNE and UMAP clustering.
+
+## Usage
+
+This notebook is intended to be run via the command line from the top directory
+of the repository as follows:
+
+```
+Rscript -e "rmarkdown::render('analyses/molecular-subtyping-HGG/09-HGG-with-braf-clustering.Rmd', clean = TRUE)"
+```
+
+## Set up
+
+### Libraries and functions
+
+```{r}
+# Load in tidyverse functions
+library(tidyverse)
+```
+
+### Directories
+
+```{r}
+# File path to scratch directory -- this contains the cleaned data
+scratch_dir <- file.path("..", "..", "scratch")
+
+# File path to data directory
+data_dir <- file.path("..", "..", "data")
+
+# File path to plots directory -- this contains the multipanel output plots
+plots_dir <- file.path("plots")
+
+if (!dir.exists(plots_dir)) {
+  dir.create(plots_dir)
+}
+
+```
+
+### Functions
+
+```{r}
+# Source the functions for dimension reduction
+source(
+  file.path(
+    "..",
+    "transcriptomic-dimension-reduction",
+    "util",
+    "dimension-reduction-functions.R"
+  )
+)
+```
+
+#### Function to join metadata with HGG molecular subtype file
+
+```{r}
+align_data <- function(aligned_scores_df,
```

Comment:
Couple ideas here:
* Could you join using the column that contains the RNA-seq biospecimen IDs instead? Which I think should be `Kids_First_Biospecimen_ID` in the aligned metadata.
* If so - I think you're only interested in biospecimens that are in both the aligned scores and in the molecular subtype data and can use an inner join.
* Filter _first_